components/integration/replace-apollo-graphql/code/apollo-graphql-bbe/better-security.md

```diff
@@ -0,0 +1,50 @@
+---
+title: 'Better security'
+description: 'Ballerina provides robust security features such as encryption, authentication, and authorization, which are essential for businesses dealing with sensitive data.'
+url: 'https://github.com/anupama-pathirage/ballerina-scenarios/tree/main/ballerina-zapier-samples/gdrive-new-event-to-slack-message'
```

Comment:
I changed the example for security common blade to have HTTP service. https://github.com/anupama-pathirage/ballerina-scenarios/blob/main/ballerina-zapier-samples/github-summary-via-secure-service/main.bal But for graphql page can we have new graphql service example with some security construct instead of this one?

Reply:
Updated with 13c6c94aa68f7344aa95ee4fb140d3d6991e170f